src/Covid19Dashboard/IllinoisMapChart/index.jsx

```diff
@@ -82,65 +93,118 @@ class IllinoisMapChart extends React.Component {
     };
   }
 
+  setSliderDates = (id) => {
+    let workingId = id;
+    if (workingId === 'time') {
+      workingId = 'C';
+    }
+    function msToDays(ms) {
+      return Math.floor(ms / (1000 * 60 * 60 * 24));
+    }
+    const curentDateRange = this.state.dataDateRange[workingId];
+    const startDate = new Date(curentDateRange.min);
+    const endDate = new Date(curentDateRange.max);
+
+    const dateDiff = msToDays(endDate - startDate);
+    let sliderValue;
+    let { sliderDate } = this.state;
+    if (sliderDate) {
+      // match date when switching between data sets
+
+      // TODO convert sliderDate to date object
```

Comment:
Is this TODO still needed?

Reply:
ill remove the todo as it was merely a note for myself